<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="styleSheet">
    <string notr="true">.QLineEdit,.QTableWidget,.QHeaderView,.QTextEdit,.QPushButton{	
	background-color:rgba(255,255,255,0.5);
	border-radius:3px;
	color: black;
}

.QLineEdit::disabled{
	background-color:rgba(155,155,155,0.5);
}

.QHeaderView{
	background-color:rgba(255,255,255,0.2);
}


.QLineEdit:focus,.QTextEdit:focus{
	background-color:rgba(255,255,255,0.6);
}

.QPushButton::hover{
	background-color:rgba(255,255,255,0.8);
}

.QFrame{
	background-color:transparent;
	border:none;
}

#mainWidget{
/*	background-image:url(:/images/link.jpg);*/
	border-image: url(:/images/link.jpg) 0 0 0 0 stretch stretch;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="processingFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="inputBox">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;I HAS A loli ITZ &amp;quot;kawaii&amp;quot;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;VISIBLE loli&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;loli R &amp;quot;not kawaii&amp;quot;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;VISIBLE loli&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="analyzeButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Analyze</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="outputBox">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputLine">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="tableFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="lexemeTable">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Lexeme</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Token</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="symbolTable">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Data Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionAnalyze"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionLOLi"/>
    <addaction name="actionAuthors"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/toolbar/document-open.svg</normaloff>:/toolbar/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/toolbar/document-save.svg</normaloff>:/toolbar/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/toolbar/quit.png</normaloff>:/toolbar/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="text">
    <string>&amp;Analyze</string>
   </property>
  </action>
  <action name="actionLOLi">
   <property name="text">
    <string>&amp;LOLi</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>&amp;Authors</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
